code/lightna/lightna-engine/App/Layout.php

```diff
@@ -49,7 +49,6 @@ public function block(string $blockName = '', array $vars = []): string
         $this->current[] = $block = $this->resolveBlock($blockName);
```

Comment:
this one also needs to be tested with fpc_compatible = true, in this case ids are used more and a bit differently

Reply:
seems fine, but please re-check